src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3263,7 +3263,16 @@ protected override IPointerTypeSymbol CommonCreatePointerTypeSymbol(ITypeSymbol
             var parameterTypesWithAnnotations = parameterTypes.SelectAsArray(
                 type => TypeWithAnnotations.Create(type.EnsureCSharpSymbolOrNull(nameof(parameterTypes)), type.NullableAnnotation.ToInternalAnnotation()));
 
-            return FunctionPointerTypeSymbol.CreateFromParts(returnTypeWithAnnotations, returnRefKind, parameterTypesWithAnnotations, parameterRefKinds, this).GetPublicSymbol();
+            // PROTOTYPE(func-ptr): Allow passing in calling convention and ref custom modifiers for return/parameters
```

Comment:
>ref custom modifiers [](start = 76, length = 20)

"ref custom modifiers"?

Reply:
What's the question?

---
In reply to: [457680708](https://github.com/dotnet/roslyn/pull/46110#discussion_r457680708) [](ancestors = 457680708)

Reply:
> What's the question?

Why only ref modifiers, it looks like we are not allowing to pass any modifiers

---
In reply to: [457779909](https://github.com/dotnet/roslyn/pull/46110#discussion_r457779909) [](ancestors = 457779909,457680708)